.gitignore

```diff
@@ -4,10 +4,11 @@ node_modules/
 db.sqlite3
 *.pyc
 .DS_Store
+.env
 
 apps/frontend/static/
 apps/guide/settings/local.py
 
 /media/
 /static/
-.coverage
\ No newline at end of file
+.coverage
```

Comment:
Cna you re-add the end of line?

Reply:
This one I edited in vim, which removes new-lines apparently